collections-csharp/UpdateItemInListCSharp/src/UpdateItemInListCSharp.Test/LibraryTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿using UpdateItemInListCSharp.Models;
+
+namespace UpdateItemInListCSharp.Test
+{
+    public class LibraryTest
+    {
+        [Fact]
+        public void WhenCheckoutBookFind_ThenBookIsCheckedOut()
+        {
+            // Arrange
+            var library = new Library();
```

Comment:
Library can be created as a private readonly property of a class. 
In each method we can create one new Book object with IsCheckedOut property set to false and add that object in the library collection. Then we can checkout that book using different methods and finally verify the IsCheckedOut property of the new book object

Reply:
I had always taken the first book from the list because I had not implemented a GetBookByIsbn() method. To identify the IsCheckedOut property of the books, should I implement this method in the Library class in each Assert section of the tests? Then we can always create new books and compare them at the end of each test.

Reply:
What I was suggesting is as follows:
1. Create new Book object in each test case with IsCheckedOut=false.
2. Then add that book object in the list
3. Next checkout the same book which we created in step 1
4. Finally in Assert verify IsCheckedOut=true or not.